applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_strategy.h

```diff
@@ -820,7 +822,7 @@ class MPMStrategy
                 Element::Pointer pelem;
 
                 // FindPointOnMesh find the element in which a given point falls and the relative shape functions
-                bool is_found = SearchStructure.FindPointOnMesh(xg, N, pelem, result_begin);
+                bool is_found = SearchStructure.FindPointOnMesh(xg, N, pelem, result_begin, MaxNumberOfResults, Tolerance);
```

Comment:
why do you change this?
This doesn't have to do with the tests

Reply:
I changed this because suppose that I want to specify the tolerance of my search. Not using the current default.

Reply:
I am planning to separate the functionalities in the strategy in the upcoming PR for the coupling. This includes the SearchElement function

Reply:
fine for me, but out of experience this can be a sensible change